mon-pix/app/components/warning-banner.js

```diff
@@ -1,8 +1,31 @@
-import Component from '@ember/component';
-import classic from 'ember-classic-decorator';
+import { htmlSafe } from '@ember/string';
+import Component from '@glimmer/component';
+const _ = require('lodash');
 import ENV from 'mon-pix/config/environment';
 
-@classic
 export default class WarningBanner extends Component {
-  isEnabled = ENV.APP.IS_WARNING_BANNER_ENABLED;
+  bannerType = ENV.APP.WARNING_BANNER_TYPE;
+
+  _bannerContent = ENV.APP.WARNING_BANNER_CONTENT;
```

Comment:
Avec le renommage du component ça donnerait 
```suggestion
  bannerType = ENV.APP.COMMUNICATION_BANNER_TYPE;

  _bannerContent = ENV.APP.COMMUNICATION_BANNER_CONTENT;
```